node/actors/bft/src/leader/replica_commit.rs

```diff
@@ -103,7 +103,11 @@ impl StateMachine {
             .expect("Could not add message to CommitQC");
 
         // Calculate the CommitQC signers weight.
-        let weight = self.config.genesis().committee.weight(&commit_qc.signers);
+        let weight = self
+            .config
+            .genesis()
+            .validators_committee
```

Comment:
let's rename it back to "validators" then.

Reply:
I deleted the prefix from the attesters too.